live-site/components/judging-section/judging-selection/JudgingSelection.styles.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+import { H2 } from '../../../../shared-ui/style/typography';
+
+const StyledJudgingSelection = styled.div`
+  padding: 10em;
+  justify-content: center;
+`;
+
+const StyledWhoAreYou = styled(H2)`
```

Comment:
For these type of things I like to keep them a bit more general like StyledJudgingWelcomeHeader in case the actual text ends up changing! But that's just a nitpick - up to you!